strongbox-web-core/src/main/java/org/carlspring/strongbox/forms/configuration/security/ldap/LdapConfigurationTestForm.java

```diff
@@ -0,0 +1,52 @@
+package org.carlspring.strongbox.forms.configuration.security.ldap;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+public class LdapConfigurationTestForm
+{
+
+    @NotEmpty
+    private String username;
```

Comment:
this probably should be renamed to `testUsername` and `testPassword` for not to confuse people that this is just to test how LDAP authentication works

Reply:
there is `@JsonProperty` annotation which can map this field to any UI capable JSON property name.

Reply:
In that case, if you guys agree that's a more clear property naming - it's fine by me :+1: :)